README.md

```diff
@@ -2,7 +2,25 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
 
 ## Getting Started
 
-First, run the development server:
+First, clone the repo and cd into your directory
+
+```bash
+git clone https://github.com/fintual-oss/paraffin-front.git && cd paraffin-front
+```
+
+Create a `.env.local` file and ask someone for the ENV variables
+
+```bash
+touch .env.local
```

Comment:
hay que agregar tambien la variable `NEXT_PUBLIC_BACK_URL` con la URL en localhost que apunta al servidor del backend (url del servidor de rails en local)

Reply:
The content of the `.env.local` should include these variables:
```
NEXT_PUBLIC_BACK_URL=
NEXT_PUBLIC_FRONT_URL=
```

Reply:
Included the variables in the readme without the values